test/scenarios/el92-src@upgrade-rpm-ok.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Sourced from cleanup_scenario.sh and uses functions defined there.
+
+scenario_create_vms() {
+    prepare_kickstart host1 kickstart.ks.template ""
+    launch_vm host1
+}
+
+scenario_remove_vms() {
+    remove_vm host1
+}
+
+scenario_run_tests() {
```

Comment:
This implementation is not going to produce junit.xml output. If the test is that we can install the RPM, it needs to be done in an RF test. If the test is that the system can run, the RPM should be added to the image via a blueprint for consistency with the other scenarios and then an RF test should be run to verify the installed system.